### DISCUSSION TITLE
SvelteKit access to user info and session updates

### DISCUSSION THREAD

**Discussion by coldino** [nextauthjs/next-auth#6182] [Help]
Got the SvelteKit systrem up and running nice and easily on a test project with GitHub OAuth and Upstash Redis in place. Works great 👍 

However, now I have a user logged in how to do get access to something like a user ID so I can do useful things like store data associated with it?

Also, am I right in thinking there's no way to update a session in the current SvelteKit setup, even on the server side?

**Comment by coldino** [comment]
Any hints on this please? I'm struggling to see how the system can be used at all as-is.

**Comment by oskarrough** [comment]
If I understand you correctly, you just want the user/session info. For that, see https://authjs.dev/reference/sveltekit/modules/main#managing-the-session.
